go/arrow/ipc/ipc.go

```diff
@@ -168,6 +169,26 @@ func WithDictionaryDeltas(v bool) Option {
 	}
 }
 
+// WithMinSpaceSavings specifies a percentage of space savings for
+// compression to be applied to buffers.
+//
+// Space savings is calculated as (1.0 - compressedSize / uncompressedSize).
+//
+// For example, if minSpaceSavings = 0.1, a 100-byte body buffer won't
+// undergo compression if its expected compressed size exceeds 90 bytes.
+// If this option is unset, compression will be used indiscriminately. If
+// no codec was supplied, this option is ignored.
+//
+// Values outside of the range [0,1] are handled as errors.
+//
+// Note that enabling this option may result in unreadable data for Arrow
+// C++ versions prior to 12.0.0.
```

Comment:
Doesn't this also apply to go? (Or else, why did we have problems in the integration test if this only affected C++ readers?)

Reply:
gah, my fault for copying the comment, i'll fix it and update.